remote-method/USAGE

```diff
@@ -0,0 +1,6 @@
+Description:
+  Creates a new remote method in the LoopBack application.
+
+Example:
+
+  slc loopback:remote-method
```

Comment:
Can we alias the command? ie. `slc loopback:remote-method` calls `slc loopback:method`, best to have both IMO. We can document `slc loopback:remote-method` as the official way and for experts, they can look up the shortcuts.